docs/customization/fixtures.rst

```diff
@@ -256,33 +206,40 @@ in ``App\Entity\Factory\ShippingMethodExampleFactory``:
         }
     }
 
-**4** Here we create ``config/packages/fixtures.xml`` and override our services:
+**3.** Then, in the ``config/services.yaml`` configure our services:
 
-.. code-block:: xml
+.. code-block:: yaml
 
-    <?xml version="1.0" encoding="UTF-8"?>
+    sylius.fixture.example_factory.shipping_method:
+        class: App\Fixture\Factory\ShippingMethodExampleFactory
+        arguments:
+            - "@sylius.factory.shipping_method"
+            - "@sylius.repository.zone"
+            - "@sylius.repository.shipping_category"
+            - "@sylius.repository.locale"
+            - "@sylius.repository.channel"
+            - "@sylius.repository.tax_category"
+        public: true
+
+    sylius.fixture.shipping_method:
+        class: App\Fixture\ShippingMethodFixture
+        arguments:
+            - "@sylius.manager.shipping_method"
+            - "@sylius.fixture.example_factory.shipping_method"
+        tags:
+            - { name: sylius_fixtures.fixture }
+
+.. tip::
 
-    <container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
-        <services>
-            <defaults public="true" />
+    When creating fixtures services manually, remember to turn off autowiring for them:
```

Comment:
I don't think it's necessary (these fixtures can be fully autowired and autoconfigured with Sylius 1.5+).

Reply:
But we are in the 1.4 branch ;>